bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/handler/MiIoBasicHandler.java

```diff
@@ -160,6 +160,9 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                         cmd = cmd + "[" + preCommandPara1 + "\"" + command.toString().toLowerCase() + "\"" + para + "]";
                     } else if (paramType == CommandParameterType.ONOFFPARA) {
                         cmd = cmd.replace("*", command.toString().toLowerCase()) + "[]";
+                    } else if(paramType == CommandParameterType.ONOFFBOOL) {
+                        boolean boolCommand = command == OnOffType.ON;
+                        cmd = cmd + "[" + preCommandPara1 + "\"" + boolCommand + "\"" + para + "]";
```

Comment:
trying to understand the difference for this and the one in https://github.com/openhab/openhab2-addons/blob/bec32e869aae165dc36ad5ceab4ff106c87e0daf/bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/handler/MiIoBasicHandler.java#L159

Reply:
It's the same, but it sends "true"/"false" instead of "on"/"off". Which is required for this device.